fdbserver/TenantCache.actor.cpp

```diff
@@ -188,26 +190,26 @@ class TenantCacheImpl {
 	ACTOR static Future<Void> monitorStorageQuota(TenantCache* tenantCache) {
 		TraceEvent(SevInfo, "StartingTenantCacheStorageQuotaMonitor", tenantCache->id()).log();
 
-		state Transaction tr(tenantCache->dbcx());
+		state Reference<ReadYourWritesTransaction> tr = tenantCache->dbcx()->createTransaction();
 
 		loop {
 			try {
-				state RangeResult currentQuotas = wait(tr.getRange(storageQuotaKeys, CLIENT_KNOBS->TOO_MANY));
+				tr->setOption(FDBTransactionOptions::READ_SYSTEM_KEYS);
+				state KeyBackedRangeResult<std::pair<TenantGroupName, int64_t>> currentQuotas =
+				    wait(TenantMetadata::storageQuota().getRange(tr, {}, {}, CLIENT_KNOBS->TOO_MANY));
```

Comment:
Can `TOO_MANY` be replaced by `MAX_TENANTS_PER_CLUSTER`?

Reply:
Yeah, that seems reasonable